client/components/jetpack/backup-card/index.tsx

```diff
@@ -0,0 +1,185 @@
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import { Card } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import ActivityActor, { SIZE_S } from 'calypso/components/activity-card/activity-actor';
+import ActivityDescription from 'calypso/components/activity-card/activity-description';
+import ActivityMedia from 'calypso/components/activity-card/activity-media';
+import Badge from 'calypso/components/badge';
+import useGetDisplayDate from 'calypso/components/jetpack/daily-backup-status/use-get-display-date';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import Tooltip from 'calypso/components/tooltip';
+import useTrackCallback from 'calypso/lib/jetpack/use-track-callback';
+import { backupDownloadPath, backupRestorePath } from 'calypso/my-sites/backup/paths';
+import getDoesRewindNeedCredentials from 'calypso/state/selectors/get-does-rewind-need-credentials';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+import cloudIcon from 'calypso/components/jetpack/daily-backup-status/status-card/icons/cloud-success.svg';
+
+/**
+ * Type dependencies
+ */
+import type { Activity } from 'calypso/state/activity-log/types';
+
+type Props = {
+	activity: Activity;
+	subActivities?: Activity[];
+	isLatest: boolean;
+	isFeatured: boolean;
+};
+
+const BackupCard: FunctionComponent< Props > = ( {
+	activity,
+	subActivities,
+	isLatest,
+	isFeatured,
+} ) => {
+	const { activityTs, activityTitle, rewindId } = activity;
+
+	const translate = useTranslate();
+	const getDisplayDate = useGetDisplayDate();
+	const moment = useLocalizedMoment();
+
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const siteSlug = useSelector( ( state ) => getSelectedSiteSlug( state ) ) || '';
+	const hasCredentials = useSelector(
+		( state ) => ! getDoesRewindNeedCredentials( state, siteId )
+	);
+
+	const restoreRef = useRef( null );
+	const [ isTooltipVisible, setTooltipVisibility ] = useState( false );
+	const onRestoreButtonEnter = useCallback( () => setTooltipVisibility( true ), [
+		setTooltipVisibility,
+	] );
+	const onRestoreButtonLeave = useCallback( () => setTooltipVisibility( false ), [
+		setTooltipVisibility,
+	] );
+	const onDownloadClick = useTrackCallback( noop, 'calypso_jetpack_backup_download', {
+		rewind_id: rewindId,
+	} );
+	const onRestoreClick = useTrackCallback( noop, 'calypso_jetpack_backup_restore', {
+		rewind_id: rewindId,
+	} );
+
+	return (
+		<Card
+			className={ classNames( 'backup-card', {
+				'is-featured': isFeatured,
+			} ) }
+		>
+			<div className="backup-card__main">
+				<div className="backup-card__header">
+					<div className="backup-card__header-text">
+						<h2 className="backup-card__date">
+							<time dateTime={ moment( activityTs ).format() }>
+								{ getDisplayDate( activityTs, false ) }
+							</time>
```

Comment:
I think maybe this is a bug that slipped through from the last iteration, but I'm noticing not that the displayed time seems incorrect.  For example, see screenshot, I'm getting "Today 2:40pm" when it should be, "Today 7:40am".
![Screenshot on 2020-10-20 at 08-23-27](https://user-images.githubusercontent.com/11078128/96608025-cb09ee80-12ad-11eb-87d5-ed3997051c4f.png)

I notice in the markup that the `dateTime` attribute seems to have the correct time though..  Here:

![Markup 2020-10-20 at 08 17 08](https://user-images.githubusercontent.com/11078128/96608242-fdb3e700-12ad-11eb-9c1b-692ec86948bd.png)

And when I reload the page sometimes I get a quick flash of the correct time (7:40am), and then it re-renders the incorrect time (2:40pm).

Reply:
The time is displayed in the timezone of the site (UTC+0 by default for a JN site). If that's what you're using to test, in your example it would make sense to display 2:40pm, which is UTC time at your 7:40am. Is what you're seeing matching prod?

Reply:
Ahhh yep! It's using timezone of the site! 👍   So it is correct.  Yes, I see it's that way in prod too, so yeah, makes sense now. Ok Cool, thanks for clearing that up.  Sounds good! 🙂